sys/sys/imgact_elf.h

```diff
@@ -39,7 +39,8 @@
 
 #define	AUXARGS_ENTRY(pos, id, val) \
     {(pos)->a_type = (id); (pos)->a_un.a_val = (val); (pos)++;}
-#if __has_feature(capabilities) && !defined(__ELF_CHERI)
+#if (__has_feature(capabilities) && !defined(__ELF_CHERI)) || \
+    (defined(_LP64) && __ELF_WORD_SIZE == 32)
```

Comment:
I think we generally prefer `__LP64__`.  I removed `_LP64` from arch(7) a while back in preference to the other spelling.

Reply:
That's what I get for trusting arch(7) on 12.1. :)